extensions/positron-duckdb/src/test/extension.test.ts

```diff
@@ -260,7 +260,12 @@ suite('Positron DuckDB Extension Test Suite', () => {
 						ExportFormat.Html
 					]
 				},
-				convert_to_code: { support_status: SupportStatus.Unsupported }
+				convert_to_code: {
+					support_status: SupportStatus.Supported,
+					code_syntaxes: [{
+						code_syntax_name: "SQL"
```

Comment:
by default, it doesn't seem like there is syntax highlighting available unless you have an extension with a sql lsp